frontend/src/app/index.tsx

```diff
@@ -1,37 +1,13 @@
-import React from "react";
-import { SafeAreaView } from "react-native";
-import { StatusBar } from "expo-status-bar";
-import { Link, Stack } from "expo-router";
+import React from 'react';
+import { View } from 'react-native';
 import { HomeScreen } from "../screens/HomeScreen";
-import sampleUser from "@/mock-data/userNickDoan";
+import { sampleUser } from "@/mock-data/user";
 
 export default function App() {
-  // return (
-  //   <View className="flex-1 bg-white items-center justify-center">
-  //     <Text>Open up app/index.tsx to start working on your app!</Text>
-  //     <Link href='/trips/2'>Go to trip detail page</Link>
-  //     <Link href='/trips/661f78b88c72a65f2f6e49d4'>Go to trip at the beach</Link>
-  //     <Link href='/trips/create'>Go to trip create</Link>
-  //     <StatusBar style="auto" />
-  //     {/* <App /> */}
-  //   </View>
-
-  // );
 
   return (
-    <SafeAreaView className="flex-1 bg-white">
-      <Stack.Screen
-        options={{
-          title: "",
-          headerShown: false,
-        }}
-      />
+    <View className="flex-1 bg-white">
       <HomeScreen user={sampleUser} />
-      <Link href="/trips/661f78b88c72a65f2f6e49d4">
-        Go to trip at the beach
-      </Link>
-      <Link href="/trips/create">Go to trip create</Link>
-      <StatusBar style="auto" />
-    </SafeAreaView>
+    </View>
```

Comment:
Ahh in HomeScreen.tsx I used SafeAreaView instead of wrapping it in the index.tsx file

index.tsx SafeAreaView + homescreen SafeAreaView or index.tsx View + homescreen Safearea
![image](https://github.com/trihoang0809/tourific/assets/89886131/17518ea9-46f5-4272-b60d-fa056b0917b8)

homescreen View + index SafeArea
![image](https://github.com/trihoang0809/tourific/assets/89886131/7ade23e2-819c-44b8-b74e-25c2eb5af976)